include/openssl/ssl.h.in

```diff
@@ -1511,6 +1511,19 @@ const char *SSL_group_to_name(SSL *s, int id);
 int SSL_set0_tmp_dh_pkey(SSL *s, EVP_PKEY *dhpkey);
 int SSL_CTX_set0_tmp_dh_pkey(SSL_CTX *ctx, EVP_PKEY *dhpkey);
 
+typedef enum {
```

Comment:
enums in public headers aren't good. They can break ABI compatibility. If you need the constants, they have to be #defined.

Reply:
Moved to ssl/ssl_local.h